### PR TITLE
Added a starting Security Policy

### PR DESCRIPTION
**Pull request by tmuntaner**
## Description

Let's avoid users creating security issues in github and instead point
them to the SUSE Security Team.

## Change Type

*Please select the correct option.*

- [x] **Documentation Update** (a change which only updates documentation)

## Checklist

*Please check off each item if the requirement is met.*

- [x] I have verified that my code follows RMT's coding standards with `rubocop`.
- [x] I have reviewed my own code and believe that it's ready for an external review.
- [x] I have provided comments for any hard-to-understand code.
- [x] I have documented the `MANUAL.md` file with any changes to the user experience.
- [x] RMT's test coverage remains at 100%.
- [x] If my changes are non-trivial, I have added a changelog entry to notify users at `package/obs/rmt-server.changes`.

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Unless this is a trivial change, please include a CHANGELOG entry.<br />
Run <code>osc vc</code> in the <code>package</code> directory to add one.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by ngetahun — APPROVED** [review]
LGTM!

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/128204/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged d063615f0a8eca4f240ade38d06d074ac2a93b37 into 76a5d95df80154dce699c815ad992ffb4d576338)

File: docs/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+Currently, only the latest version of RMT is supported. Whenever we release a new version of RMT, we submit it to all supported versions of SLE and openSUSE which support RMT.
+
+## Reporting a Vulnerability
+
+**DO NOT CREATE AN ISSUE** to report a security problem. Instead, please contact the [SUSE Security Team](https://www.suse.com/support/security/contact/) instead.
```
Comment (lagartoflojo) [inline docs/SECURITY.md:9]: > Instead, ... instead.

😉

Comment (tmuntaner) [reply docs/SECURITY.md:9]: Lol, if you don't mind, I made you a reviewer in the followup pr: https://github.com/SUSE/rmt/pull/609

**Pull request merged**